stdlib/asyncio/events.pyi

```diff
@@ -80,6 +80,7 @@ _Context: TypeAlias = dict[str, Any]
 _ExceptionHandler: TypeAlias = Callable[[AbstractEventLoop, _Context], Any]
 _ProtocolFactory: TypeAlias = Callable[[], BaseProtocol]
 _SSLContext: TypeAlias = bool | None | ssl.SSLContext
+_TaskFactory: TypeAlias = Callable[[AbstractEventLoop, Coroutine[Any, Any, _T] | Generator[Any, None, _T]], Future[_T]]
```

Comment:
Pyright now thinks that we want this to be a generic type alias. We can fix this by using a callback protocol instead:

```suggestion

class _TaskFactory(Protocol):
    def __call__(self, __loop: AbstractEventLoop, __factory: Coroutine[Any, Any, _T] | Generator[Any, None, _T]) -> Future[_T]: ...
```